rebar.config

```
@@ -37,7 +37,9 @@
             {branch, "master"}
         }
     },
-    {damsel,         {git, "git@github.com:rbkmoney/damsel.git"               , {branch, "release/erlang/master"}}},
+    {damsel,         {git, "git@github.com:rbkmoney/damsel.git"               ,
+        {branch, "release/erlang/epic/payment_methods_refactoring"}}
```

Кажется уже пора

```suggestion
        {branch, "release/erlang/master"}}
```

?

---

Оно не в мастере пока, так и лежит в эпик ветке. Не вливал т.к. это потребовало бы довольно быстро катнуть кучу вещей

---

> потребовало бы довольно быстро обновить все сервисы

:thinking: А можешь пояснить почему?

---

Перечитал то, как работает thrift, понял что сказал глупость. Тогда замержу в мастер и подтяну где надо